.github/workflows/release-build.yml

```diff
@@ -0,0 +1,106 @@
+# This workflow creates a new tag, builds the release archives and uploads them to GitHub and our server
+# It is triggered by pushing to the stable branch, either manually or by merging a PR created by the
+# release-preparation workflow
+
+name: "Release: Tag, Build & Deploy"
+on:
+  push:
+    branches:
+      - stable
+
+jobs:
+
+  tag:
+    name: Tag Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Prepare Environment
+        run: |
+          php .github/release.php current >> $GITHUB_ENV
+
+      - name: Check if a tag already exists
+        run: |
+          if git rev-parse "release-${{ env.current_version }}" >/dev/null 2>&1; then
+            echo "::error::Tag already exists, be sure to update the VERSION file for a hotfix"
+            exit 1
+          fi
```

Comment:
so only new tag if VERSION is changed? from intro it was for me not clear when it is triggered.

Reply:
Correct. The changed version file is the trigger to create a new tag and release. So when doing a manual hotfix release, you can cherry-pick a commit or manually push a commit and only then change the VERSION file in an additional commit to trigger the build.